Source/Chronozoom.UI/ui/auth-edit-timeline-form.html

```diff
@@ -30,13 +30,32 @@
     <div class="cz-form-time-end" style="display: block"></div>
 
     <br />
-    <label class="cz-lightgray cz-form-label">End Time *</label>
-    <div class="cz-form-time-end" style="display: block"></div>
+    <label class="cz-lightgray cz-form-label cz-form-offset-label">
+        Top boundary (<span title="Distance to child timeline top measured in percents of parent timeline height" class="cz-form-percent-tooltip">%</span>)
```

Comment:
Bad description. Better to be "Timeline top position measured in percent of parent timeline height"